samples/Playground/Playground.iOS/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Plugin.Permissions" version="5.0.0-beta" targetFramework="xamarinios10" />
```

Comment:
Please don't use old approach for add nuget, use `<PackageReference />` in csproj